types/minio/index.d.ts

```diff
@@ -176,6 +176,55 @@ export interface LegalHoldOptions {
     status: LegalHoldStatus;
 }
 
+export interface InputSerialization {
+    CompressionType?: string;
+    CSV?: {
+        AllowQuotedRecordDelimiter?: boolean;
+        Comments?: string;
+        FieldDelimiter?: string;
+        FileHeaderInfo?: string;
+        QuoteCharacter?: string;
+        QuoteEscapeCharacter?: string;
+        RecordDelimiter?: string;
+    };
+    JSON?: {
+        [key: string]: any;
```

Comment:
Looks like `JSON` takes an optional `Type` property and nothing else: https://docs.aws.amazon.com/AmazonS3/latest/API/API_JSONInput.html

Reply:
Fixed